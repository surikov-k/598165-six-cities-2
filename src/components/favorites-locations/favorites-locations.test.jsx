import React from 'react';
import renderer from 'react-test-renderer';
import FavoritesLocations from './favorites-locations';

jest.mock(`react-router-dom`,
    () => ({
      Link: () => null,
    }));

describe(`<FavoritesLocations/>`, () => {
  it(`renders correctly`, () => {
    const component = renderer
      .create(<FavoritesLocations
        city=""
        places={[]}
        onChangeFavorite={() => {}}
      />)
      .toJSON();
    expect(component).toMatchSnapshot();
  });
});

