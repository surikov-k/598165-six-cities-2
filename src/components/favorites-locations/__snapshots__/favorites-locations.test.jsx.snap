// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FavoritesLocations/> renders correctly 1`] = `
<li
  className="favorites__locations-items"
>
  <div
    className="favorites__locations locations locations--current"
  >
    <div
      className="locations__item"
    />
  </div>
  <div
    className="favorites__places"
  />
</li>
`;
